packages/server/lib/config.js

```diff
@@ -44,194 +58,6 @@ const removeEnvPrefix = (key) => {
   return key.slice(CYPRESS_ENV_PREFIX_LENGTH)
 }
 
-const folders = toWords(`\
-fileServerFolder   fixturesFolder   integrationFolder   pluginsFile
-screenshotsFolder  supportFile      supportFolder       unitFolder
-videosFolder\
-`)
-
-// for experimentalComponentTesting
-folders.push('componentFolder')
-
-// Public configuration properties, like "cypress.json" fields
-const configKeys = toWords(`\
-animationDistanceThreshold      fileServerFolder
-baseUrl                         fixturesFolder
-blockHosts
-chromeWebSecurity
-modifyObstructiveCode           integrationFolder
-env                             pluginsFile
-hosts                           screenshotsFolder
-numTestsKeptInMemory            supportFile
-port                            supportFolder
-projectId                       videosFolder
-reporter
-reporterOptions
-ignoreTestFiles
-testFiles                       defaultCommandTimeout
-trashAssetsBeforeRuns           execTimeout
-userAgent                       pageLoadTimeout
-viewportWidth                   requestTimeout
-viewportHeight                  responseTimeout
-video                           taskTimeout
-videoCompression
-videoUploadOnPasses
-screenshotOnRunFailure
-watchForFileChanges
-waitForAnimations               resolvedNodeVersion
-nodeVersion                     resolvedNodePath
-firefoxGcInterval
-retries
-`)
-
-// NOTE: If you add a config value, make sure to update the following
-// - cli/types/index.d.ts (including allowed config options on TestOptions)
-// - cypress.schema.json
-
```

Comment:
can you add this note to config_options.ts ? Since we do not yet generate typedefs

Reply:
Good call.